src/Service.Tests/SqlTests/RestApiTests/Insert/MySqlInsertApiTests.cs

```diff
@@ -182,6 +182,20 @@ public void InsertOneInViewBadRequestTest()
             throw new NotImplementedException();
         }
 
+        [TestMethod]
+        [Ignore]
+        public override Task InsertOneWithComputedFieldMissingInRequestBody()
```

Comment:
Why is mysql excluded?

Reply:
Because we don't collect metadata about read only fields for MySql. And hence the term ComputedFieldMissing/included does not make sense right now.